mockOSApi.v2/Aux/Errors.cs

```diff
@@ -0,0 +1,12 @@
+namespace mockOSApi.Errors;
+
+public class ErrorMessages
+{
+
+    public Dictionary<string, string> ErrorMessage { get; set; } = new Dictionary<string, string>();
```

Comment:
Generally, if you want Static Error messages, you would use a Static Class with Consts, like this

```cs
public static class ErrorMessages {
     public const string PathError = "The path does not exist or could not be found";
}
```

At a large scale though, you would use a Localization library

Reply:
Implemented Localization error messages in Services/ErrorMessage.cs, will remove this file and keep using that for further references